src/objective.jl

```diff
@@ -8,6 +8,35 @@
 # See http://github.com/JuliaOpt/JuMP.jl
 #############################################################################
 # This file contains objective-related functions
+"""
+    simplex_iterations(model::Model)
+Return the cumulative number of simplex iterations after a call to `optimize!(model)`.
+Exact value depends upon implementation of MathOptInterface.SimplexIterations()
```

Comment:
Missing backquotes surounding the MOI attributes. Also add a link to the MOI doc